tests/aws/services/transcribe/test_transcribe.py

```diff
@@ -1,28 +1,120 @@
+from localstack.services.transcribe.provider import LANGUAGE_MODELS, TranscribeProvider
 from localstack.testing.pytest import markers
 from localstack.utils.files import new_tmp_file
 from localstack.utils.strings import short_uid, to_str
 from localstack.utils.sync import poll_condition, retry
+from localstack.utils.threads import start_worker_thread
 
 BASEDIR = os.path.abspath(os.path.dirname(__file__))
 
 LOG = logging.getLogger(__name__)
 
+# Lock and event to ensure that the installation is executed before the tests
+INIT_VOSK_LOCK = threading.Lock()
+INIT_FFMPEG_LOCK = threading.Lock()
+vosk_installed = threading.Event()
+ffmpeg_installed = threading.Event()
+installation_errored = threading.Event()
+
+INSTALLATION_TIMEOUT = 5 * 60
+PRE_DOWNLOAD_LANGUAGE_CODE_MODELS = ["en-GB"]
+
+
+def install_async():
+    """
+    Installs the default ffmpeg and vosk versions in a worker thread.
+    """
+    if vosk_installed.is_set() and ffmpeg_installed.is_set():
+        return
+
+    def install_vosk(*args):
+        with INIT_VOSK_LOCK:
+            if vosk_installed.is_set():
+                return
+            try:
+                LOG.info("installing Vosk default version")
+                vosk_package.install()
+                LOG.info("done installing Vosk default version")
+                LOG.info(
+                    "downloading Vosk models used in test: %s", PRE_DOWNLOAD_LANGUAGE_CODE_MODELS
+                )
+                for language_code in PRE_DOWNLOAD_LANGUAGE_CODE_MODELS:
+                    model_name = LANGUAGE_MODELS[language_code]
+                    # downloading the model takes quite a while sometimes
+                    TranscribeProvider.download_model(model_name)
+                    LOG.info(
+                        "done downloading Vosk model '%s' for language code '%s'",
+                        model_name,
+                        language_code,
+                    )
+                LOG.info("done downloading all Vosk models used in test")
+            except Exception:
+                LOG.exception("Error during installation of Vosk dependencies")
+                installation_errored.set()
+                # we also set the other event to quickly stop the polling
+                ffmpeg_installed.set()
+            finally:
+                vosk_installed.set()
+
+    def install_ffmpeg(*args):
+        with INIT_FFMPEG_LOCK:
+            if ffmpeg_installed.is_set():
+                return
+            try:
+                LOG.info("installing ffmpeg default version")
+                ffmpeg_package.install()
+                LOG.info("done ffmpeg default version")
+            except Exception:
+                LOG.exception("Error during installation of Vosk dependencies")
+                installation_errored.set()
+                # we also set the other event to quickly stop the polling
+                vosk_installed.set()
+            finally:
+                ffmpeg_installed.set()
```

Comment:
The locking here shouldn't be necessary in my opinion, because the package installers already handle the locking.
(You could even have fine-grained locking control with the installers, see optional `install_lock` kwarg on the `PackageInstaller` class).

Reply:
I removed, will wait for the green pipeline and run it once again after just to be sure 👌